editor/templates/index.mst

```diff
@@ -87,13 +87,13 @@
 
 <div id="node-dialog-confirm-deploy" class="hide">
     <form class="form-horizontal">
-        <div id="node-dialog-confirm-deploy-config" style="text-align: left; padding-top: 30px;" data-i18n="[prepend]deploy.confirm.improperlyConfigured;[append]deploy.confirm.confirm">
-            <ul style="font-size: 0.9em; width: 400px; margin: 10px auto; text-align: left;" id="node-dialog-confirm-deploy-invalid-list"></ul>
+        <div id="node-dialog-confirm-deploy-config" class="node-dialog-confirm-deploy-config" data-i18n="[prepend]deploy.confirm.improperlyConfigured;[append]deploy.confirm.confirm">
+            <ul class="node-dialog-confirm-deploy" style="" id="node-dialog-confirm-deploy-invalid-list"></ul>
```

Comment:
`node-dialog-confirm-deploy` is already used as the id of the dialog - not suitable to reuse is as the css style name for the `<ul>`